docs/ReleaseNewVersion.md

```diff
@@ -30,10 +28,9 @@ Once the new release is ready, you should add the latest release to the `Moodle
 1. Login to the plugins' directory site and go to [plugin page](https://moodle.org/plugins/local_codechecker).
 2. To add a new version, you have to be a plugin maintainer. **Only lead maintainers can add other maintainers**.
 3. In the main action bar, follow the *</>Developer zone* option.
-4. Zip the code in a codechecker folder and name it "local_codechecker_moodle39_2020100200.zip" (being "2020100200" the plugin
- version). Don't forget to remove the .git folder.
+4. Donwload a zip file for the given version of the plugin from GitHub (https://github.com/moodlehq/moodle-local_codechecker/archive/v2.9.8.zip).
```

Comment:
Ah, this is good (download for later upload).

I just was saying that the plugins database, when adding a new version, automatically looks @ github and shows you a dropdown to pick any version released directly, instead of having to download and upload the .zip.

![pick_directly_from_github](https://user-images.githubusercontent.com/167147/96861114-efa7c700-1463-11eb-89cb-04f510a48f33.png)

And it's nice because it saves you from those manual steps.

Reply:
It's not nice... it's amazing!! :D I had no idea. Thank you!

Reply:
Say thanks to @mudrd8mz !